helpdesk_timesheet/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Helpdesk Timesheet",
+    "summary": """
+        Add Timesheets to the Helpdesk""",
+    "category": "Helpdesk",
```

Comment:
`helpdesk_mgmt` is categorized as 'After-Sales'

Reply:
btw why was the name changed?

Reply:
Do you mean the module name? It was because there's an enterprise module already named `helpdesk`

Reply:
Yes exactly! Im not sure if we have to rename our modules depending on enterprise modules.
Enterprise is private and we cannot know anythoing from that. So i generally disagree with that move :D

Reply:
Of course we cannot know if tomorrow there'll be another module called `helpdesk_mgmt` or if it exists a module called `potato` but we have to bear in mind that this is still Odoo.Mmany users of OCA are probably using Enterprise Edition and that means we have to minimize all possible errors.

Reply:
yes i know your idea. maybe this is also not the place for the discussion :)
but basically i would ignore enteprise completely a name modules for what they are.
here: helpdesk.